### DISCUSSION TITLE
Existe ferramenta de análise de pull request sem precisar entrar no github?

### DISCUSSION THREAD

**Discussion by renedocket** [frontendbr/forum#1923] [Perguntas e Respostas]
Olá, pessoal! Tudo bem?

Eu gostaria de saber se você tem ferramentas pra indicar sobre análise de PR, sem precisar entrar no Github? 
Essa minha dúvida, é justamente pra ter um parâmetro se já existe algo desse tipo pra não precisar desenvolver uma solução que talvez resolva meu problema que é, não ter que ir no site do github pra ver o PR saca?

Desde já, muito abrigado a todos! :)

**Comment by giovannism20** [comment]
Você quer ver os PR's do repo sem precisar entrar ou quer testar o PR localmente antes de aceitar e dar o merge ?

**Comment by renedocket** [comment]
@GiovanniSM20 então eu gostaria de ver o pr, se tivesse comentários e afim no pr conseguir ver através da ferramente, é uma coisa mais visual mesmo. Acredito que não necessitária ter o merge, apenas o acompanhamento mesmo do Pr. Não sei se deu pra entender rs

**Comment by vitebo** [comment]
Eu costumo utilizar o [cli do github](https://cli.github.com/) pra essas coisas.

**Comment by giovannism20** [comment]
@renedocket Se você usa VSCode também tem essa extensão: https://github.com/Microsoft/vscode-pull-request-github
